modules/apps/josh-lapurd-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir='<@liferay.language key="lang.dir" />' lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+
+	<#if is_signed_in>
+		<style>
+			#jw-banner {
+				margin-top: -30px !important;
+			}
+
+			#banner {
+				height: 80px !important;
+			}
+
+			#jw-nav-bar {
+				float: right;
+				top: 65px !important;
+			}
+
+			.jumbotron-picture {
+				top: -25px !important;
+			}
+		</style>
+	</#if>
+
+	<@liferay_util["include"] page=top_head_include />
+</head>
+
+<body class="${css_class}">
+
+<@liferay_ui["quick-access"] contentId="#main-content" />
+
+<@liferay_util["include"] page=body_top_include />
+
+<@liferay.control_menu />
+
+<div class="container-fluid" id="wrapper">
+	<header id="banner" role="banner">
+		<div class="row" id="jw-banner">
+			<div class="col-md-2 col-sm-2 col-xs-2 jw-nav-bar jw-nav-left" id="company-branding">
+				<div class="site-name">
+					<a href="localhost:8080/" rel="home" title="Home">${site_name}</a>
+				</div>
+			</div>
+			<nav id="login">
+				<div class="float-right">
+					<#if !is_signed_in>
+						<a class="link-button" data-redirect="${is_login_redirect_required?string}" href="${sign_in_url}" id="sign-in" rel="nofollow"><div id="sign_in_text">${sign_in_text}</div></a>
+					</#if>
+					<a class="float-right jw-nav-bar-toggle">&#x2261;</a>
+				</div>
+
+				<!-- Dynamic navigation menu -->
+				<div class="float-right" id="jw-nav-bar">
+
+					<#if has_navigation && is_setup_complete>
+						<a class="jw-nav-bar-toggle">&#x2261;</a>
+						<#include "${full_templates_path}/navigation.ftl" />
+					</#if>
+
+				</div>
+				<!-- /Dynamic navigation menu -->
+
+			</nav>
+		</div>
+	</header>
+
+	<section id="hero">
+		<div class="row">
+			<div class="jumbotron-picture">
+				<div class="background-box">
+					<nav id="breadcrumbs">
+						<@liferay.breadcrumbs />
+					</nav>
+				</div>
+			</div>
+		</div>
+	</section>
+
+	<section id="content">
+		<div class="theme-container">
+			<h1 class="hide-accessible">${the_title}</h1>
+			<div class="row">
+				<div class="col-md-12 text-center">
+					<h1>This is my <span style="color: #DAB485">Lapurd</span> theme</h1>
+					<p>
+					Donec consectetur mauris justo, vel scelerisque nunc dictum sed. Ut sodales arcu sed bibendum egestas. Proin lorem lacus, venenatis id enim at, sagittis eleifend odio. Ut porta blandit metus ac molestie. Ut lacinia nibh vitae felis porttitor, a maximus tortor sollicitudin. Aliquam sit amet sapien ac tellus efficitur vestibulum. Nulla elementum porta ante et facilisis.
+					</p>
```

Comment:
We probably should have hard coded HTML and text in the ```portal_normal.ftl``` file.  This should be part of the theme or web content?